app/main/index.ts

```diff
@@ -234,7 +238,7 @@ app.on('ready', () => {
 	});
 
 	ipcMain.on('clear-app-settings', () => {
-		global.mainWindowState.unmanage(mainWindow);
+		globalPatched.mainWindowState.unmanage();
```

Comment:
We haven't passed the window here. Is this intentional?

Reply:
This was wrongly declared before. We don't need to pass anything to unmanage here. Passing of mainWindow is only require is a manage call.

Reply:
You can refer to [this](https://github.com/mawie81/electron-window-state/blob/master/index.js).

Reply:
Got it, thanks.